index.html

```diff
@@ -322,6 +322,36 @@ <h3 class="skills_dropdown_title dropdown_child_class">
                   </li>
                </ul>
            </section>
+
+           <section class="contact" id="contact">
+            <div class="contact__container">
+              <h2 class="contact_header">Contact me</h2>
+              <p class="contact_description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+              </p>
+              <form action="#" method="post">
```

Comment:
- [ ] As pointed by the requirement, kindly add your Formspree link to the action so you can collect the data sent by the user, and please make sure that you receive the input data there when testing it
 
![image](https://user-images.githubusercontent.com/73140461/148840575-bbe7c33a-603b-4ef8-8903-6b9ea7f52df7.png)